Automate/RedHatConsulting_Satellite6/Infrastructure/VM/Retirement/StateMachines/VMRetirement.class/default.yaml

```diff
@@ -13,6 +13,8 @@ object:
       on_entry: update_retirement_status(status => 'Unregistering from Satellite'')
       on_exit: update_retirement_status(status => 'Unregistered from Satellite'')
       on_error: update_retirement_status(status => 'Failed Unregistering from Satellite'')
+  - ReleaseIPAddress:
+      value: "\t/Infrastructure/Network/Operations/Methods/ReleaseIPAddress"
```

Comment:
What's the deal with the \t in value? There's also one in the retirement state machine

Reply:
copy paste issue, needs fixing.